articles/node-vs-python/index.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: engineering-education
+status: publish
+published: true
+slug: node-vs-python
+title: APIs in Node.js vs Python - A Comparison
+description: Node.js and Python are among the most popular languages used to write APIs and backend services. Let's look at a comparison between the two most popular languages out there.
+author: saiharsha-balasubramaniam
+date: 2020-08-13T00:00:00-17:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/node-vs-python/hero.jpg
+    alt: node.js vs Python
+---
+JavaScript and Python are the most popular languages among web developers. The idea of separating the front-end from the back-end as much as possible has birthed the idea of the JAMStack. JAMStack stands for **JavaScript, APIs and Markup**. It describes the idea of a website being completely static and client-side rendered. The website uses API calls to load all dynamic content. Let us look at the two most popular languages for writing APIs, Node and Python to compare and contrast the two.
+<!--more-->
+
+### Table of Contents
+
+- [The History](#the-history)
+- [Node vs Python: The Differences](#node-vs-python:-the-differences)
+- [APIs in Node](#apis-in-node)
```

Comment:
These two links no longer match the headings so the links are broken.

Reply:
Thanks a ton for suggesting these changes. I'll work on them and commit my changes!